music21/languageExcerpts/instrumentLookup.py

```diff
@@ -1404,49 +1308,46 @@
 
         Note: No length test due to duplicate entries
-        (i.e. allToBestName is smaller than the sum of its parts).
+        (i.e. allToClassName is smaller than the sum of its parts).
         '''
-        for eachDict in [abbreviationToBestName,
-                         englishToBestName,
-                         frenchToBestName,
-                         germanToBestName,
-                         italianToBestName,
-                         russianToBestName,
-                         spanishToBestName,
-                         transliteration]:
+        for eachDict in [abbreviationToClassName,
+                         englishToClassName,
+                         frenchToClassName,
+                         germanToClassName,
+                         italianToClassName,
+                         russianToClassName,
+                         spanishToClassName]:
 
             for key in eachDict:
-                self.assertIn(key, allToBestName)
+                self.assertIn(key, allToClassName)
 
-    def testAllToBestNameExamples(self):
+    def testAllToClassNameExamples(self):
         '''
-        Test an example from each constituent dict that makes up allToBestName.
+        Test an example from each constituent dict that makes up allToClassName.
         '''
-        abbreviationTest = 'a sax'
-        self.assertEqual(allToBestName[abbreviationTest], abbreviationToBestName[abbreviationTest])
 
-        englishTest = 'accordion'
-        self.assertEqual(allToBestName[englishTest], englishToBestName[englishTest])
+        for testString, langDict in [('a sax', abbreviationToClassName),
+                                     ('accordion', englishToClassName),
+                                     ('accord\xe9on', frenchToClassName),
+                                     ('aeolophon', germanToClassName),
+                                     ('a becco', italianToClassName),
+                                     ("al't", russianToClassName),
+                                     ('acorde\xf3n', spanishToClassName)
+                                     ]:
+            self.assertEqual(allToClassName[testString], langDict[testString])
 
-        frenchTest = 'accord\xe9on'
-        self.assertEqual(allToBestName[frenchTest], frenchToBestName[frenchTest])
-
-        germanTest = 'aeolophon'
-        self.assertEqual(allToBestName[germanTest], germanToBestName[germanTest])
-
-        italianTest = 'a becco'
-        self.assertEqual(allToBestName[italianTest], italianToBestName[italianTest])
-
-        russianTest = "al't"
-        self.assertEqual(allToBestName[russianTest], russianToBestName[russianTest])
-
-        spanishTest = 'acorde\xf3n'
-        self.assertEqual(allToBestName[spanishTest], spanishToBestName[spanishTest])
+    def testAllClassNames(self):
+        '''
+        Test that all class names are real.
+        '''
+        from music21 import instrument as instr
+        for v in allToClassName.values():
```

Comment:
Nitpick detail, but can these tests use self.subTest? Some examples in the codebase. So we don't have to re-run to get all the failures.